docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  builder:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: builder:${TAG-local}
+    container_name: ubuntu-builder
+    stdin_open: true
+    tty: true 
+    environment: 
+      - TZ='Europe/London'
+    volumes:
+      - '/etc/timezone:/etc/timezone:ro'
+      - '/etc/localtime:/etc/localtime:ro'
+      - ${CUSTOM_SSH_KEY-~/.ssh/id_rsa}:/root/.ssh/id_rsa:ro
```

Comment:
Note: I wanted to follow up on these mounts. If the files don't exist on the host then docker will create them as directories which causes a lot of bother and isn't the desired behaviour. I need to research if there is a better way to manage this or get the process to fail if these files (or their overrides) don't exist on the host.

Reply:
With the new script, this is actually handled so happy with that.